src/components/experimental/IndexTable/IndexTable.scss

```diff
@@ -0,0 +1,320 @@
+@import '../../../styles/common';
+
+$index-table-stacking-order: (
+  cell: 1,
+  sticky-cell: 30,
+  sticky-table: 31,
+  scroll-bar: 35,
+  bulk-actions: 36,
+  loading-panel: 37,
+);
+
+$breakpoint-small: 458px;
+
+.IndexTable {
+  position: relative;
+  overflow: hidden;
+}
+
+.LoadingContainer-enter {
+  opacity: 0;
+  transform: translateY(-100%);
+}
+
+.LoadingContainer-enter-active {
+  opacity: 1;
+  transition: opacity duration(fast) ease-out, transform duration(fast) ease-out;
+  transform: translateY(0);
+}
+
+.LoadingContainer-exit {
+  opacity: 1;
+  transform: translateY(0);
+}
+
+.LoadingContainer-exit-active {
+  opacity: 0;
+  transform: translateY(-100%);
+  transition: opacity duration(fast) ease-in, transform duration(fast) ease-in;
+}
+
+$loading-panel-height: rem(53px);
+
+.LoadingPanel {
+  position: absolute;
+  z-index: z-index(loading-panel, $index-table-stacking-order);
+  top: 0;
+  left: 0;
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  height: $loading-panel-height;
+  background: color('sky', 'lighter');
+  box-shadow: shadow();
+}
+
+.Table {
+  width: 100%;
+  min-width: 100%;
+  border-collapse: collapse;
+}
+
+.Table--scrolling {
+  .TableCell--first,
+  .TableCell--second,
+  .TableHeading--first,
+  .TableHeading--second {
+    visibility: visible;
+    background-color: color('white');
```

Comment:
I think this background colour should be added even without scrolling. If you scroll fast you can see the text behind the cell and then disappears when the class is applied.